Sources/XMTP/Messages/MessageV1.swift

```diff
@@ -82,7 +82,7 @@ extension MessageV1 {
 
 	var sentAt: Date {
 		// swiftlint:disable force_try
-		try! Date(timeIntervalSince1970: Double(header.timestamp) / 1_000_000)
+		try! Date(timeIntervalSince1970: Double(header.timestamp / 1000))
```

Comment:
Just to double check: Is this correct here? I would assume the header timestamp is is in nanoseconds here and that the timeIntervalSince1970 would be milliseconds.

Reply:
Related (and feel free to ignore!) but we ended up adding nanosecond / millisecond conversion helpers in both `xmtp-js` and [Flutter](https://github.com/xmtp/xmtp-flutter/pull/10/files#diff-ad10af43adb21e81d72e92a82c6bd81c04ed9dca3c18cacbd585cd38d50c0e7fR6) in case this inspires you or is helpful at all 😆

Reply:
🤦 My test was wrong here, once I used the millisecond helper I had already added ages ago, I was able to remove this. Great catch!